competencies/roles/_TechWideGeneral.md

```diff
@@ -43,7 +43,12 @@
         </ul></td>
     </tr>
     <tr>
-        <td>
+        <td colspan="3">
+            Refer to the notes on <b>Interpreting Business Acumen</b> below for clarification
```

Comment:
Notes for business acumen are in a separate [PR](https://github.com/GoProperly/organization/pull/21)